### ISSUE TITLE
Issue in Set-SFVolumeAccessGroup example 2

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#28]
```
  -------------------------- EXAMPLE 2 --------------------------

    PS C:\>$inits = Get-SFInitiator -Alias ''
    Set-SFVolumeAccessGroup -VolumeAccessGroupID 1 -Initiators $inits

    Adds all initators with no alias to VolumeAccessGroup 1.
```

The issue is $inits is object.

```
PS C:\Program Files\SolidFire> $inits = Get-SFInitiator -Alias 'templater'
PS C:\Program Files\SolidFire> echo $inits


Alias              : templater
InitiatorID        : 3
InitiatorName      : iqn.1991-05.com.microsoft:win16
VolumeAccessGroups : {}
Attributes         : {}
```

In this case `Set-SFVolumeAccessGroup -VolumeAccessGroupID 1 -Initiators $inits` creates a VAG with a new and illegal initiator name.

**Comment by ahaid** [comment]
Good catch. I have created bug SDK-869 to address this.

**Comment by Salmanius** [comment]
This has been fixed in the help and will be included in the 1.6 release

**Comment by arjun960** [comment]
This issue has been fixed, so closing the issue.
